$NetBSD: patch-src_corelib_Qt5CoreConfigExtras.cmake.in,v 1.1 2015/05/18 14:02:02 nros Exp $
Then PIE is set cmake sometimes builds object files that are to be used
in shared objects with PIE when is supposed to use only PIC.
This patch exists in qt5 5.5.
http://code.qt.io/cgit/qt/qtbasegit/log/src/corelib/Qt5CoreConfigExtras.cmake.in
--- src/corelib/Qt5CoreConfigExtras.cmake.in.orig	2015-05-12 21:00:11.000000000 +0000
+++ src/corelib/Qt5CoreConfigExtras.cmake.in
@@ -72,7 +72,7 @@ set(_qt5_corelib_extra_includes)
 # macro to add it.
 set(Qt5_POSITION_INDEPENDENT_CODE True)
 set_property(TARGET Qt5::Core PROPERTY INTERFACE_POSITION_INDEPENDENT_CODE \"ON\")
-set(Qt5Core_EXECUTABLE_COMPILE_FLAGS \"-fPIE\")
+set(Qt5Core_EXECUTABLE_COMPILE_FLAGS \"-fPIC\")
 !!ENDIF
 
 !!IF !isEmpty(QT_NAMESPACE)
